packages/aragon-cli/test/guides/customDeploy.test.js

```diff
@@ -0,0 +1,286 @@
+import test from 'ava'
+
+import {
+  runAragonCLI,
+  getLocalWeb3,
+  matchAddressAtLineContaining,
+} from '../utils'
+
+// NOTE: These tests validate that the guide at https://hack.aragon.org/docs/guides-custom-deploy can be followed without errors and as expected.
+
+// TODO: These tests simply assure that there is no error thrown when running commands from the guide. Additionally, some assertions could be made on each test.
+
+// Enable verbose to debug/develop the test, but disable for production.
+const verbose = false
+
+let ACCOUNT1, ACCOUNT2, DAO, ACL, TOKEN, TOKEN_MANAGER, VAULT, FINANCE, VOTING
+
+test.before(async t => {
+  // Identify accounts
+  const web3 = await getLocalWeb3()
+  const accounts = await web3.eth.getAccounts()
+  ACCOUNT1 = accounts[0]
+  ACCOUNT2 = accounts[1]
+  if (verbose) console.log(`ACCOUNT1`, ACCOUNT1)
+  if (verbose) console.log(`ACCOUNT2`, ACCOUNT2)
+})
+
+test.serial('creates a dao', async t => {
```

Comment:
Consider using `import { serial as test } from 'ava'` and then `test('...'`
I find it cleaner and easier to reason about: you know all tests are serial.